.github/workflows/self-scheduled-intel-gaudi3-caller.yml

```diff
@@ -0,0 +1,70 @@
+name: Self-hosted runner (Intel Gaudi3 scheduled CI caller)
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "17 2 * * *"
+  push:
+    branches:
+      - gaudi-ci
+
+jobs:
+  model-ci:
+    name: Model CI
+    uses: ./.github/workflows/self-scheduled-intel-gaudi.yml
+    with:
+      job: run_models_gpu
```

Comment:
So this will run all the model tests right?

Reply:
when I remove the line https://github.com/huggingface/transformers/pull/38790/files#diff-8c37d32b57986fbdc245587445b81e5c48582b8b489b46de168f8f4efb0d28baR65, should I do one run with all models in this PR ?

Reply:
too late 😅, but now we know that it takes [3h 14m 48s](https://github.com/huggingface/transformers/actions/runs/15772176919/usage) to run the entire thing ! not bad, even great (given the inefficiency of re-pulling the images and re-installing the deps)